graphql-dgs-platform/build.gradle.kts

```diff
@@ -51,14 +51,12 @@ dependencies {
         api("com.graphql-java:graphql-java-extended-scalars") {
             version { require(Versions.GRAPHQL_JAVA_EXTENDED_SCALARS) }
         }
+        api("com.graphql-java:graphql-java-extended-validation") {
+            version { require(Versions.GRAPHQL_JAVA_EXTENDED_VALIDATION) }
+        }
         api("com.apollographql.federation:federation-graphql-java-support") {
             version { require(Versions.GRAPHQL_JAVA_FEDERATION) }
         }
-        api("com.graphql-java:graphql-java-extended-validation"){
-            // The version below will work with Jakarta EE 8
```

Comment:
Why remove the comments?

Reply:
Ok, they were added to the version file.

Reply:
I don't see much value on keeping the versions in the version file. Ot is an unnecessary level of indirection. I'll address this in a different pr.

Reply:
comments moved to `Versions.kt`